02-peer-protocol.md

```diff
@@ -2721,12 +3365,30 @@ The sending node:
   - if it has sent `commitment_signed` for an interactive transaction construction but
     it has not received `tx_signatures`:
     - MUST set `next_funding_txid` to the txid of that interactive transaction.
+    - if it has not received `commitment_signed` for that interactive transaction:
+      - MUST set `next_commitment_number` to the commitment number of the `commitment_signed` it sent.
   - otherwise:
     - MUST NOT set `next_funding_txid`.
+  - if `option_splice` was negotiated:
+    - if a splice transaction reached acceptable depth while disconnected:
+      - MUST include `my_current_funding_locked` with the txid of the latest such transaction.
+    - otherwise:
+      - MUST include `my_current_funding_locked` with the txid of the last `splice_locked` it sent.
+      - if it never sent `splice_locked` for any transaction, but it sent `channel_ready`:
+        - MUST include `my_current_funding_locked` with the txid of the channel funding transaction.
```

Comment:
```
      - if it has sent `splice_locked` for any transaction:
        - MUST include `my_current_funding_locked` with the txid of the last `splice_locked` it sent.
      - otherwise:
        - if it sent `channel_ready`:
          - MUST include `my_current_funding_locked` with the txid of the channel funding transaction.
        - otherwise (it has never sent `channel_ready` or `splice_locked`):
          - MUST NOT include `my_current_funding_locked`.
```

Reply:
Thanks, this is indeed easier to understand and translate into code, done in https://github.com/lightning/bolts/pull/1160/commits/70a2047fe49966cc91762d992ab2547576f8ac3e (where I also fixed the indenting of the next requirements which was wrong).